src/core/core.config.js

```diff
@@ -99,6 +99,21 @@ function mergeConfig(...args/* config objects ... */) {
 	});
 }
 
+function includePluginDefaults(options) {
+	options.plugins = options.plugins || {};
+	options.plugins.title = (options.plugins.title !== false) && merge(Object.create(null), [
```

Comment:
Should this be more generic? What if other plugins have defaults?

Reply:
those are actually handled in core.plugins in generic way, just our internal plugins are not using it :)